install-libraries.sh

```diff
@@ -0,0 +1 @@
+pip3 install -r install-libraries.txt
```

Comment:
does this work? the filename is `install_libraries.txt`

Reply:
Filenames were not matching but all names updated to use "_" instead of "-" and this works now.